project/iterative_sorting.py

```diff
@@ -32,16 +32,22 @@ def insertion_sort(arr):
     return result
 
 
-print(insertion_sort([5, 3, 2, 2, 4, 6, 9]))
-
-
 # STRETCH: implement the Bubble Sort function below
 def bubble_sort(arr):
+    result = arr[:]
```

Comment:
nice! a few other ways to do it here I just found
https://stackoverflow.com/questions/2612802/how-to-clone-or-copy-a-list